subprojects/packagefiles/x-plane-sdk/meson.build

```diff
@@ -0,0 +1,121 @@
+project(
+    'x-plane-sdk', 'c', 'cpp',
```

Comment:
From a quick glance I didn't see a C compiler being used for anything (other than library lookups that could be done with a C++ compiler just as easily). Do we need both?

Reply:
The library, except for the C++ wrappers, is exposed as C. Am I mistaken to mark it as having C, in that case?

Reply:
Those parts appear to be precompiled SDK binaries?

You don't need a C compiler to link to C libraries, only to compile them. In theory, someone that has a C++ compiler installed but doesn't have a C compiler (what *does* that mean, exactly??? who knows) could link to precompiled libraries in their C++ only software.

Reply:
I've updated the build script -- a couple fixes, as well as making building the C++ wrappers optional, since not all may use them (instead using [PPL](https://github.com/PhilippMuenzel/PPL) if they still use C++).

This way, only people who really want the included wrappers get them, and it only asks for a C++ compiler if using them. Granted, the only platforms X-Plane supports all have C++ compilers, so I would be concerned if one managed to end up without one, but I suppose for minimal installs on Linux?